### ISSUE TITLE
Test of accessible login creation interface

### ISSUE THREAD

**Issue by avneeshsingh** [daisy/epub-accessibility-tests#25]
The first two tests in basic test sections state the following:
Basic Functionality Tests
[REQUIRED]file-010

The device can be started independently and essential accessibility for starting and exiting applications is available.

→Indicate Pass or Fail.

1

[REQUIRED]file-110

The reading system can be activated independently.

→Indicate Pass or Fail.

If we look from an app perspective, both of the tests are asking the same question.

One way can be to update the first test to evaluate the accessibility of login interface of reading apps.

for example:
Basic Functionality Tests
[REQUIRED]file-010

If you are using a hardware device, it can be started independently and essential accessibility for starting and exiting applications is available.

If you are using reading app that requires you to create an online account for getting started then the complete getting started process should be accessible.

→Indicate Pass or Fail.

1

**Comment by prashantverma2014** [comment]
Agree that the first tests should be modified to include accessibility of the log in interface etc. 

Similarly, in the third test we can add comments about testing the accessibility of book download process in case of browser based apps. Many apps do not allow side loading of files. We have been getting our file inside the app with the help of developers etc. Actually the objective of this test is to check if users can search, select and download files from the reading system if it has or is connected to online libraries.  This should be clarified in the third test. 

[REQUIRED]file-310

Open content: Open the test book in the Reading System.

Indicate Pass or Fail.

**Comment by avneeshsingh** [comment]
Created new issue for this: #26.
